src/AvatarStack.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {get, COMMON} from './constants'
+import theme from './theme'
+
+const rightStyles = `
+  .body {
+    right: 0;
+    flex-direction: row-reverse;
+
+    &:hover .avatar {
+      margin-right: 0;
+      margin-left: 3px;
+    }
+  }
+
+  .avatar.avatar-more {
+    background: ${get('colors.gray.3')};
+
+    &::before {
+      width: 5px;
+    }
+
+    &::after {
+      width: 2px;
+      background: ${get('colors.gray.1')};
+    }
+  }
+
+  .avatar {
+    margin-right: 0;
+    margin-left: -11px;
+    border-right: 0;
+    border-left: ${get('borders.1')} solid ${get('colors.white')};
+  }
+`
+
+const Item = styled.img.attrs(() => ({
+  className: 'avatar'
+}))`
+  position: relative;
+  z-index: 2;
+  display: flex;
+  width: 20px;
+  height: 20px;
+  box-sizing: content-box;
+  margin-right: -11px;
+  background-color: ${get('colors.white')};
+  border-right: ${get('borders.1')} solid ${get('colors.white')};
```

Comment:
I think you can remove the `solid`. It looks like `borders.1` is `1px solid` and this is causing this CSS rule to be `border-right: 1px solid solid #fff` and the white border is missing:

![image](https://user-images.githubusercontent.com/5487287/66207102-e0e74a00-e67f-11e9-9434-06b1f7270ea3.png)

```suggestion
  border-right: ${get('borders.1')} ${get('colors.white')};
```

Reply:
ahhh good catch!